.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
 .todo
 *.mp4
-*.egg-info
\ No newline at end of file
+*.egg-info
+*.vscode
```

Comment:
please blank line at the end of the file

Reply:
done